pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -128,4 +128,13 @@ under the License.
             <resolver>org.apache.hawq.pxf.plugins.gemfirexd.GemFireXDResolver</resolver>
         </plugins>
     </profile>
+    <profile>
+        <name>Json</name>
+        <description>A profile for JSON data, one JSON record per line</description>
```

Comment:
looks like we've consolidate 2 profile - should update the description to not limit it to "one JSON record per line".